src/black/const.py

```diff
@@ -1,4 +1,23 @@
+from enum import Enum
+from typing_extensions import TypedDict
+
 DEFAULT_LINE_LENGTH = 88
 DEFAULT_EXCLUDES = r"/(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.svn|_build|buck-out|build|dist)/"  # noqa: B950
 DEFAULT_INCLUDES = r"(\.pyi?|\.ipynb)$"
 STDIN_PLACEHOLDER = "__BLACK_STDIN_FILENAME__"
+
+
+class Empty(TypedDict):
+    pass
+
+
+class LogLevel(Enum):
+    none: Empty = {}
```

Comment:
```suggestion
    none: Dict[str, Any] = {}
```
How about simply this instead?

Reply:
But it isn't entirely correct though.